src/component/calculator.js

```diff
@@ -1,44 +1,77 @@
 /* eslint-disable react/prop-types */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable max-classes-per-file */
 import React from 'react';
+import calculate from '../logic/calculate';
 
-function Result({ result }) {
-  return (
-    <div className="display-section">
-      <p className="diplay-value">{result}</p>
-    </div>
-  );
+class Calculator extends React.Component {
```

Comment:
- [ ] Kindly remove these linter disable lines as we need linter checks to meet standards. Your component is supposed to be a functional component anyways and should use the `useState` function, instead of `this.state`

Reply:
Thank you for your feedback. I will remove the linter disable lines and switch from using `this.state` to `useState` in my functional component to meet coding standards.